src/components/GoodsList.jsx

```diff
@@ -0,0 +1,9 @@
+import { Good } from './Good';
+
+export const GoodsList = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <Good key={good} good={good} />
+    ))}
```

Comment:
it is semantically better to wrap your`Good` into `li` tag inside list component